src/System.Security.Cryptography.X509Certificates/tests/System.Security.Cryptography.X509Certificates.Tests.builds

```diff
@@ -11,14 +11,22 @@
       <OSGroup>Unix</OSGroup>
     </Project>
     <Project Include="System.Security.Cryptography.X509Certificates.Tests.csproj">
+      <OSGroup>Unix</OSGroup>
+      <TargetGroup>netcoreapp1.1</TargetGroup>
```

Comment:
No, the .csproj says if no targetgroup was defined to be netstandard1.7 (line 15)